index.html

```diff
@@ -120,28 +120,45 @@ <h2 class="title">About Me</h2>
     <div class="underline"><img src="media/underline_dark-backgroud.svg" alt=""></div>
   </section>
   <section class="skills-section">
-    <div class="title-languages"><img src="media/Icons/Rectangle-55.png" alt="">   Languages</div>
+    <h4 class="title-languages"><img src="media/Icons/Rectangle-55.png" alt="">   Languages</h4>
     <ul class="skills">
       <li>Javascript</li>
       <li>Ruby</li>
       <li>HTML</li>
       <li>CSS</li>
     </ul>
-    <div class="title"><img src="media/Icons/Rectangle-56.png" alt="">   Frameworks</div>
+    <h4 class="title"><img src="media/Icons/Rectangle-56.png" alt="">   Frameworks</h4>
     <ul class="skills">
       <li>Bootstrap</li>
       <li>Ruby on Rails</li>
       <li>RSPec</li>
       <li>CapyBara</li>
       <li>Selenium</li>
     </ul>
-    <div class="title"><img src="media/Icons/Rectangle-57.png" alt="">   Skills</div>
+    <h4 class="title"><img src="media/Icons/Rectangle-57.png" alt="">   Skills</h4>
     <ul class="skills">
       <li>Codekit</li>
       <li>Github</li>
       <li>Gitlab</li>
       <li>Terminal</li>
     </ul>
   </section>
+  <section class="contact-section">
+    <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
```

Comment:
- [ Optional ] The content is a paragraph. Thus, it would be semantically more appropriate to use a `<p>` tag instead. You can add a heading of something like `Contact Me` using `<h2>` if you would like to have a heading for the section. what do you think?